life_hc/MySQL/General/fn_noop.sqf

```diff
@@ -0,0 +1,25 @@
+/*
+	File: fn_noop.sqf
+*/
+private _fnc_doNoop = {
+	for "_i" from 0 to 10 step 1 do
+	{
+		["noop", 1] call HC_fnc_asyncCall;
+	}
+};
+private _routines = [
+	[[], _fnc_doNoop, 3600]
+];
+{
+    _x pushBack (diag_tickTime + (_x # 2));
+} forEach _routines;
```

Comment:
A bit confused as to why you're storing one element in an array and looping over it as opposed to just setting a variable for the one element instead of using an array? What's the intention of this file?